### ISSUE TITLE
Does not support .NET 4.6 Experimental?

### ISSUE THREAD

**Issue by toddbluhm** [TouchScript/TouchScript#263]
I attempted to use TouchScript in Unity 2017.1f1 with the experimental .NET 4.6 framework and C# 6 version and it gave various compatibility errors. Is this correct in that its not supported yet?

I should add that I am use this on a mac.

**Comment by valyard** [comment]
To be honest, I've never tried. I'll check and get back to you.

**Comment by valyard** [comment]
I've tested TouchScript with .NET 4.6 and on my Mac it works fine. What errors are you getting?
Are you sure they are coming from TouchScript?

**Comment by toddbluhm** [comment]
So I have come to the conclusion that it is a bug in my project and not with your lib. 

I downgraded my project to 3.5 and used the asset store version and it was still broke. So then I used one of the example scenes and it was still broke. So finally I just created a new empty project, installed the asset store version, ran the checkers scene and it works flawlessly. So I changed the new empty project to the same player settings as my old project and it still ran great. Finally I updated the new project to 4.6 and again it ran great on my iPhone. So my conclusion is that something is really screwed up in my old project. I am going to create new project and re-import everything and see if I can get it working.

Closing this for now.
